src/Benchmarks/Data/EfDb.cs

```diff
@@ -26,19 +26,33 @@ public Task<World> LoadSingleQueryRow()
             return _dbContext.World.FirstAsync(w => w.Id == id);
         }
 
-        public async Task<World[]> LoadMultipleQueriesRows(int count)
+        public async Task<World[]> LoadMultipleQueriesRows(int count, bool withTracking = false)
         {
             var result = new World[count];
 
             for (int i = 0; i < count; i++)
             {
                 var id = _random.Next(1, 10001);
-                result[i] = await _dbContext.World.FirstAsync(w => w.Id == id);
+                var world = withTracking ? _dbContext.World.AsTracking() : _dbContext.World;
+                result[i] = await world.FirstAsync(w => w.Id == id);
             }
 
             return result;
         }
 
+        public async Task<World[]> LoadMultipleUpdatesRows(int count)
+        {
+            var result = await LoadMultipleQueriesRows(count, true);
+
+            for (int i = 0; i < count; i++)
+            {
+                result[i].RandomNumber = _random.Next(1, 10001);
+            }
+
+            await _dbContext.SaveChangesAsync();
```

Comment:
Spec says the following about transactions (Test 5, Requirement 10):

> Use of transactions is acceptable but not required. If transactions are used, a transaction should only encapsulate a single iteration, composed of a single read and single write. Transactions should not be used to consolidate multiple iterations into a single operation.

I assume it will be fastest to disable transactions for all (Raw, Dapper, and EF)?